db/migrations/000006_create_environment_reservations_table.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS environment_reservations(
+    id                serial PRIMARY KEY,
+    environment_id    integer REFERENCES environments (id),
+    name              text,
```

Comment:
name I expect to be some like `swatomation run #4452` source I expect to be the sourceIP/dns that the call is coming from. Ideally we'd see reservation locks/releases with pair name/source, but maybe if something breaks or gets stale we might see `devops cleanup` and `localhost` releases or something

Reply:
Ah gotcha, thx for clarifying! 👍 

It's a bit tricky because while Jenkins has a concept of a user-friendly job run name, I don't know if GitHub actions does? The use of `name` here makes me double-take a bit, in lots of other places in Sherlock (services, environments) it's a strictly-formatted unique identifier whereas in this case it's more freeform and informational.

Reply:
yeah the comment made me pause, I think it made a bunch more sense when it was job_request.name, but either way it's mean to link the reservation to some job. just looked into GHR, looks like there def are action_name + run_number ids, hopefully we can pull those.
![Screen Shot 2021-09-17 at 12 45 23 PM](https://user-images.githubusercontent.com/85525681/133845131-1efe94e8-f05e-4260-a297-dd27eb16b46c.png)